blockchain/store_test.go

```diff
@@ -70,7 +70,7 @@ var (
 	part1       = partSet.GetPart(0)
 	part2       = partSet.GetPart(1)
 	seenCommit1 = &types.Commit{Precommits: []*types.Vote{{Height: 10,
-		Timestamp: time.Now().UTC()}}}
+		Timestamp: time.Now().Round(0).UTC()}}}
```

Comment:
maybe it would be better to define a function for this and forbid use of `time.Now` so no one accidentally forgets ? Something like `types.Now()`

Reply:
yeah that would work too.